### PR TITLE
A wide range of new libraries

### PR DESCRIPTION
**Pull request by djzort**
Form Frameworks, Web Framework-like libraries and some popular template libraries

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,11 +221,14 @@ We also recommend these lists.
 *Libraries and tools for templating*
 
 * [Text::Templates](https://metacpan.org/pod/Text::Template) - Templates with embedded perl
+* [Template::Toolkit](https://metacpan.org/release/Template-Toolkit) - Very Popular Template Processing System
+* [Template::Allow](https://metacpan.org/release/Template-Alloy) - TT2/3, HT, HTE, Tmpl, and Velocity Engine
```
Comment (bor) [inline README.md]: typo: Template::Allow

Comment (moznion) [inline README.md]: And please use "pod" instead of "release" as path of module's URL.

**Comment by moznion** [comment]
It looks nice list :+1:

File: README.md [hunk 2]
```diff
@@ -264,11 +269,33 @@ We also recommend these lists.
...
+
+* [HTML::FormFu](https://metacpan.org/pod/HTML::FormFu) - HTML Form Creation, Rendering and Validation Framework
+* [HTML::FormFu::ExtJS](https://metacpan.org/pod/HTML::FormFu::ExtJS) - ExtJS form generation from HTML::FormFu config files
+* [Catalyst::Controller::HTML::FormFu](https://metacpan.org/pod/Catalyst-Controller-HTML-FormFu) - Use HTML::FormFu in Catalyst
+* [CGI::FormBuilder](https://metacpan.org/pod/CGI-FormBuilder) - Easily generate and process stateful forms
+* [Form::Toolkit](https://metacpan.org/pod/Form-Toolkit) - A toolkit to build Data centric Forms
+* [WWW::Form](https://metacpan.org/pod/WWW-Form) - Simple and extendable module that allows developers to handle HTML form input validation and display flexibly and consistently.
+* [Form::Sensible](https://metacpan.org/pod/Form-Sensible) - A sensible way to handle form based user interface
 
 ### Middlewares
```
Comment (uzulla) [inline README.md]: Indentation is different from the Table of Contents.
And, I think should be title change from "Middlewares", If that move here.

**Comment by moznion** [comment]
I have some questions;

- Isn't Mason a template engine?
- Isn't Embperl  template engine?
- Isn't Poet a web application framework?

Could you tell me why do you classified into `Web Framework-Like` group?

File: README.md (hunk 1 above)
Comment (djzort) [inline README.md]: both fixed

File: README.md (hunk 2 above)
Comment (djzort) [inline README.md]: I dont understand this feedback

**Comment by djzort** [comment]
Agree with move Poet. Diff updated.

Mason, Embperl attach straight to mod_perl. You can't take say TT2 or Template::Alloy and run them as your mod_perl request handler. Hence they are more than a template but perhaps not as fully featured as Catalst/Mojo/etc

File: README.md (hunk 2 above)
Comment (uzulla) [inline README.md]: please check "Middlewares" Indent depth(topic level).

**Comment by uzulla** [comment]
Thanks update, merged.

**Pull request merged**